git_machete/client.py

```diff
@@ -2058,19 +2058,19 @@ def __derive_remote_and_github_org_and_repo(self, branch_used_for_tracking_data:
                     'If that is the case, you can provide repository information explicitly, via these 3 git config keys: '
                     '`machete.github.{remote,organization,repository}`\n'))
 
```

Comment:
Okay! hope this will be some sensible compromise... I'd still expect some 1 or 2 corner cases to crop up, but nothing major